src/stories/ConfirmDialog.stories.js

```diff
@@ -0,0 +1,18 @@
+import ConfirmDialog from '~/components/confirm-dialog/ConfirmDialog'
+import i18n from '~/plugins/i18n'
```

Comment:
why do you need i18n here?

Reply:
without it text on buttons is common.no and common.yes

Reply:
ok

Reply:
import i18n in ./storybook/preview.js